lib/phlex/rails.rb

```diff
@@ -0,0 +1,6 @@
+require "phlex/rails/rendering"
```

Comment:
If require `phlex` here, everything else should be auto-loaded and we won't need to use an Array require in the Gemfile.